docs/enterprise/configuring-dns.md

```diff
@@ -31,30 +31,40 @@ For TLS, your certificate should have `stackblitz.example.com` as the Common Nam
 Having trouble updating your TLS cert in StackBlitz? Follow these instructions:
 
 1. SSH into your StackBlitz host VM
+
 2. Retrieve the existing StackBlitz application certificate and key via `​kubectl get secret -o jsonpath={.data} stackblitz-tls-secret`
-3. Create a new yaml file on the host VM called new-kotsadm-tls.yaml based on the template below:
-```apiVersion: v1
```

Comment:
Fixed a formatting issue: here the `apiVersion` part was understood as a language specifier, and not as the first line of the preformatted content.